specs/rm/requirements-management-vocab.ttl

```diff
@@ -17,7 +17,7 @@ oslc_rm:
   a owl:Ontology ;
   dcterms:title "The OSLC Requirements Management(RM) Vocabulary" ;
   dcterms:description "All vocabulary URIs defined in the OSLC Requirements Management (RM) namespace."^^rdf:XMLLiteral ;
-  dcterms:source <https://github.com/oslc-op/oslc-specs/blob/master/specs/blob/master/rm/requirements-management-vocab.ttl> ;
+  dcterms:source <https://raw.githack.com/oslc-op/oslc-specs/master/specs/rm/requirements-management-vocab.ttl> ;
```

Comment:
I think it should be https://github.com/oslc-op/oslc-specs/raw/master/specs/rm/requirements-management-vocab.ttl instead.

Reply:
Agreed.